.travis.yml

```diff
@@ -0,0 +1,35 @@
+env:
+  global:
+    - CC_TEST_REPORTER_ID=
+language: node_js
+node_js:
+  - "8"
+  - "10"
+os:
+  - linux
+  - osx
+cache:
+  directories:
+  - node_modules
+before_script:
```

Comment:
You need to install elm as well

Reply:
According to the Travis build log, at least `elm-format --help` is running fine when we have only elm-format installed without the rest of the Elm toolchain. Does it show an error somewhere?

Reply:
Yeah, the error is being written to stdout for some reason.

```
"I couldn't figure out what version of Elm you are using!

You should either run elm-format from the directory containing
your elm.json (for Elm 0.19) or elm-package.json (for Elm 0.18),
or tell me which version of Elm with --elm-version=0.19 or --elm-version=0.18

"
```

Reply:
If it's reporting an error to stdout when you try to format stdin->stdout, we need to file a bug report about that..

Reply:
That might work. The latest version of elm-format supports both Elm 0.18 and Elm 0.19. It determines the version based on the presence of `elm-package.json` (0.18) or `elm.json` (0.19) in the current directory. That means strictly the current directory only - it doesn't look in parent directories. There's also a command line flag to control the Elm version but that seems like a more difficult solution for end users.

Reply:
I've been debating whether elm-format should check the elm version for each file starting at the file and traversing up directories until it finds elm.json or elm-package.json, instead of looking in the current directory.  It seems like that would probably be more correct for more cases.  What do you think?  (In any case, that would be in elm-format 0.8.2 at the earliest)

Reply:
Great to see you here @avh4. You really care about your project! You already seem to have an issue about this at https://github.com/avh4/elm-format/issues/561 so let's continue discussion there to make sure everyone can find it :) For this plugin, the most useful missing elm-format feature would be using `isatty()` (in Haskell, `queryTerminal`) to disable ANSI output when stderr is not going a terminal.

Reply:
Discussion about Elm version number is ongoing in https://github.com/Unibeautify/beautifier-elm-format/issues/2